setup.py

```diff
@@ -57,8 +58,19 @@ def finalize_options(self) -> None:
         mkpath(self.build_temp)
         copy_file(join(dirname(__file__), 'CMakeLists.txt'),
                   self.build_temp)
-        cmake = run(['cmake', '.'], check=True, stdout=DEVNULL, stderr=PIPE,
-                    cwd=self.build_temp, universal_newlines=True)
+        try:
+            cmake = run(['cmake', '.'],
+                        check=True, stdout=DEVNULL, stderr=PIPE,
+                        cwd=self.build_temp, universal_newlines=True)
+        except CalledProcessError as e:
+            print(f"\n\ncmake: Error occurred!\n\n"
+                  f"(Standard output)\n"
+                  f"{e.stdout}\n\n"
+                  f"(Standard error)\n"
+                  f"{e.stderr}\n",
+                  file=sys.stderr)
+            sys.exit(1)
+
```

Comment:
I'd prefer a re-raise here, since it'd contain the error message which has the exit code of CMake.  
```suggestion
            raise
```
The

Reply:
I did this because the stacktrace is very misleading on what the error really is, but I see your point I'll change that

Reply:
You got a point about the traceback, so how about `raise e from None`?